sdks/python/apache_beam/typehints/native_type_compatibility_test.py

```diff
@@ -122,20 +129,22 @@ def test_convert_to_beam_type(self):
 
   def test_convert_to_beam_type_with_builtin_types(self):
     if sys.version_info >= (3, 9):
-      test_cases = [('builtin dict', dict[str, int], typehints.Dict[str, int]),
-                    ('builtin list', list[str], typehints.List[str]),
-                    ('builtin tuple', tuple[str], typehints.Tuple[str]),
-                    ('builtin set', set[str], typehints.Set[str]),
-                    (
-                        'nested builtin',
-                        dict[str, list[tuple[float]]],
-                        typehints.Dict[str,
-                                       typehints.List[typehints.Tuple[float]]]),
-                    (
-                        'builtin nested tuple',
-                        tuple[str, list],
-                        typehints.Tuple[str, typehints.List[typehints.Any]],
-                    )]
+      test_cases = [
+          ('builtin dict', dict[str, int], typehints.Dict[str, int]),
+          ('builtin list', list[str], typehints.List[str]),
+          ('builtin tuple', tuple[str],
+           typehints.Tuple[str]), ('builtin set', set[str], typehints.Set[str]),
```

Comment:
Nit: weird spacing here
```suggestion
          ('builtin tuple', tuple[str], typehints.Tuple[str]),
          ('builtin set', set[str], typehints.Set[str]),
```

Reply:
Hmm yapf did something weird here for sure, changed

Reply:
Lol, the formatter really doesn't like it I guess. Well, we tried - feel free to revert/merge or to find a way to fix forward the formatting